### ISSUE TITLE
We need to revise towards a release of the Open Leadrship Framework

### ISSUE THREAD

**Issue by chadsansing** [mozilla/olm-whitepaper#33]
Collaborating with @xmatthewx, et al., we need to release a "gold" version of the map as a CMS site with a new name - the Open Leadership Framework.

I will work on the content and architecture of the site; the ask of the production team is for help with design (e.g. a banner, other images/iconography) and redirecting old pages to the new CMS site once it is ready.

Design brief submitted 3/6/18.

**Comment by chadsansing** [comment]
@xmatthewx, any sense of timeline? More March than April, or visa versa?

**Comment by xmatthewx** [comment]
# Brief 

Support required: Design. I could use help from this team and Abby to redirect from older, GH versions of the map to the new CMS site when it is ready.

Submitted: 3/6/2018
Proposed date to start work: 3/12/2018
Proposed launch date: 4/2/2018

## DARCI
 - Driver: Chad Sansing
 - Accountable: Chad Sansing
 - Responsible: Sabrina
 - Consulted: Abby Cabunoc Mayes, Zannah Marsh
 - Informed: Sarah Allen, Steph Wright, Erika Drushka

## Summary 

The Open Leadership Map is becoming the Open Leadership Framework with a release planned for late March or early April, 2018. The new version of the Framework will be a multi-page site (say, 5 pages) on the foundation's CMS platform. It needs a Global-Sprint-esque design update to differentiate it from other OLE offerings. I would like to finish with 1) a banner and 2) reusable open leadership icons for the framework's principles (understanding, sharing, participation & inclusion) and practices (design, build, empower). 

Existing content here: https://mozilla.github.io/olm-whitepaper/.

OKR(s): 
   - OKR 5 - Work Open
   - This project will provide context, copy, & content for programs supporting OKR #5's KR1 and KR 2.

**Comment by xmatthewx** [comment]
Have you been onboarded on the CMS? You can start building content directly on the site. You can work with @sabrinang on a banner graphic. We can use some existing icons to support the content, but it's probably too much to create new designs for the content. Team is currently juggling all the things. 

@sabrinang - does this fit with what's currently on your plate? Seems like a good complement to new Global Sprint event graphics.

**Comment by chadsansing** [comment]
Thank you, @xmatthewx  & @sabrinang!

I think I have had CMS on-boarding - maybe? Would Jesse have delivered it? Maybe we should find a time to review. If it is like other WYSIWYG web-authoring tools, I should be okay. Ish.

I have content that is under review by the OLE team. I will get broader review next and then start building pages - sound right?

@sabrinang, would it be best to set up a call to talk further about the design?

**Comment by sabrinang** [comment]
@chadsansing Yes let's discuss what design assets I can help with, I am available tomorrow for a call.

**Comment by chadsansing** [comment]
Starting to edit copy on this repo from OLM to OLF, 3/19/18.

**Comment by chadsansing** [comment]
Checking in, @xmatthewx & @sabrinang.

**Comment by xmatthewx** [comment]
@chadsansing - we had a team sprint last week (which will soon result in a way better cms and some awesome staff user documentation). This week, @sabrinang is off to moz retreat. 

Are you able to start posting content? Ideally, design provides an enhancement when ready but is not a blocker.

**Comment by chadsansing** [comment]
@xmatthewx, content is there, waiting on a strategy meeting in April post retreat, but also beginning some OLF-specific coaching leading up to the sprint & would love to feature any designs in materials for that, as well.
